src/pages/guides/index.md

```diff
@@ -319,7 +319,7 @@ Kindly note that this digital signature verification process comes out-of-the-bo
+| 2500  | IMS Org  | No | A maximum of 2500 event registrations (webhook or journal) can be created for an IMS Org. This quota cannot be adjusted.                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                   |
+| 10  | IMS Credential (client Id) | Yes | A maximum of 10 event registrations can be created for an IMS client-id. <br/><br/>To keep the number of registrations in check, we recommend subscribing to multiple event types in a registration. If there are a large number of event types that need to be subscribed and adding all of them in a single registration is not desirable, then consider subscribing "related" event types together in a registration. Event types can be grouped based on the domain, estimated traffic scale, development effort required to process the events, or any other factor. <br/><br/>I/O Events is designed to support a small number of registrations in a Dev Console Project. Having a small number of registrations in a Project provides better resiliency and contributes to the cost-effectiveness of the overall architecture (as it is more likely for the webhooks to receive events in batches). In addition, having a small number of registrations minimizes the development and operational overheads associated with managing the event-consumer webhooks/runtime-actions.<br/><br/>If you have use cases that require creating a large number of event registrations in a Project, then please reach out to us via [I/O Events Forum](https://experienceleaguecommunities.adobe.com/t5/adobe-i-o-events/ct-p/adobe-io-events). We'd like to understand them in more detail and we will be happy to recommend an appropriate event-subscription pattern. |
```

Comment:
Let's use developer console instead of dev console.

Also, I believe that I/O Events experience league forum (https://experienceleaguecommunities.adobe.com/t5/adobe-i-o-events/ct-p/adobe-io-events) has been archived/deprecated in favour of the generic [Adobe Developer forum](https://experienceleaguecommunities.adobe.com/t5/adobe-developer/ct-p/adobe-io), but worth confirming this once.

Reply:
1. Using full form with consistent formatting for adobe developer console now.
2. Started a thread to confirm about the consolidation of experience league forum pages.